### PR TITLE
Test branch for branch-specific vercel ENV variables

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard1**
## Summary

This branch has both of the feature flags on the vaccination page turned on, so that the backend team can have a branch to test their data with.
Vercel allows specific ENV vars per branch, so they can test deployments themselves. (This PR is also meant to test that specific Vercel functionality)

## Motivation

DX

### Governance

- [ ] Documentation is added
- [ ] Test cases are added or updated
- [ ] I've read the contributing document https://github.com/minvws/.github/blob/master/CONTRIBUTING.md
- [ ] I've read and understand the Code of Conduct https://github.com/minvws/.github/blob/master/CODE_OF_CONDUCT.md
- [ ] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License
      https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/LICENSE.txt
      and the contributor license agreement https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/CLA.md

### CODE REVIEW

File: packages/app/schema/nl/__index.json [hunk 1]
```diff
@@ -44,7 +44,9 @@
     "vaccine_administered_rate_moving_average",
     "vaccine_administered_planned",
     "vaccine_stock",
-    "vaccine_delivery_per_supplier"
+    "vaccine_delivery_per_supplier",
+    "vaccine_coverage_per_age_group",
+    "vaccine_coverage"
```
Comment (VWSCoronaDashboard8) [inline packages/app/schema/nl/__index.json:49]: I think as long as you're using feature flags, there should be no need to add these to the required list. The flags can only really work with optional properties. As soon as they supply data that data will still be validated on its shape.

**Review by VWSCoronaDashboard8 — APPROVED** [review]

File: packages/app/src/pages/landelijk/vaccinaties.tsx [hunk 2]
```diff
@@ -88,8 +90,8 @@ const VaccinationPage = (props: StaticProps<typeof getStaticProps>) => {
   const { page } = content;
 
   // TODO: put this back this when data is available
-  //const {vaccine_coverage_per_age_group} = data;
-  const vaccine_coverage_per_age_group = mockCoverageData();
+  const { vaccine_coverage_per_age_group } = data;
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/pages/landelijk/vaccinaties.tsx:93]: clean up the commented out stuff?

**Review by VWSCoronaDashboard4 — APPROVED** [review]

**Comment by VWSCoronaDashboard10** [comment]
the PR title `test branch` sounds like it shouldn't be reviewed -- is that assumption correct?

**Comment by VWSCoronaDashboard8** [comment]
This feature is enabled in another open PR so this should be obsolete now.

**Pull request closed without merging**